Source/WebKitLegacy/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ if (APPLE)
     set_target_properties(WebKitLegacy PROPERTIES LINK_FLAGS "-umbrella WebKit")
 endif ()
 
-if (MSVC)
+if (APPLE AND MSVC)
```

Comment:
Can this be true? I think `APPLE` is true on macOS, iOS, tvOS, and watchOS. So on these platforms, MSVC will not work.
https://cmake.org/cmake/help/latest/variable/APPLE.html

Reply:
Oops. Will fix. Thank you.